scripts/kubestellar

```diff
@@ -240,6 +241,24 @@ EOF
     echo "Finished augmenting root:compute for KubeStellar"
 }
 
+function ensure_space_mgt() {
+    kubectl ws root &> /dev/null
+    $SCRIPT_DIR/space-provider-config "$KUBECONFIG" "$SCRIPT_DIR/../config/space/provider_kcp.yaml"
+    if kubectl get Workspace "$space_mgt_name" &> /dev/null; then
+	echo "space-mgt workspace already exists -- using it:"
+	kubectl ws "$space_mgt_name"
+    else
+        kubectl ws create "$space_mgt_name" --enter
+    fi
+    kubectl ws create-context --overwrite
+    # create space CRDs
+    eval kubectl create -f "$SCRIPT_DIR/../config/crds/space.kubestellar.io_spaces.yaml" $verbdir
```

Comment:
Better to write this in idempotent style; use `kubectl apply` rather than `kubectl create`.